spec/fixtures/pages/ping-pong.html

```diff
@@ -1,14 +1,12 @@
 <html>
 <body>
 <script type="text/javascript" charset="utf-8">
-  const {ipcRenderer} = require('electron')
-  ipcRenderer.on('ping', function (event, id) {
-    ipcRenderer.sendTo(id, 'pong', id)
+  ipcRenderer.on('ping', function (event, payload) {
```

Comment:
`ipcRenderer` must be `undefined` here.

Reply:
it's exposed by the preload script

Reply:
`preload: path.join(fixtures, 'module', 'preload-inject-ipc.js'),`